lib/help.js

```diff
@@ -11,7 +11,14 @@ var helpjson = {
 	'submit'			: 'To submit the code to JavaAutolab and fetch the results',
 	'help'				: 'Print help manual'
 };
+/**
+* Displays the list of autolab commands along with their functions.
+* @function help
+* @param {null}
+* @return {object} object
```

Comment:
I can't see anything getting returned. It is just being called from index.js for printing the help manual.